src/revlanguage/functions/RlFunction.cpp

```diff
@@ -133,7 +133,7 @@ std::string Function::callSignature(void) const
  *       Finally, the ellipsis arguments no longer have to be last among the rules, but they
  *       are still the last arguments after processing.
  */
-bool Function::checkArguments( const std::vector<Argument>& passed_args, std::vector<double>* match_score, bool once)
+bool Function::checkArguments( const std::vector<Argument>& passed_args, std::vector<double>* match_score, std::vector<bool>& arg_mapped, bool once)
```

Comment:
Does the `arg_mapped` argument record whether the checkArguments function managed to find a correspondence between the argument type and the function signature, for the ith argument?  Maybe a comment indicating what this does?

Reply:
Sure, will add a comment.